app/client/components/payment/update/CurrentPaymentDetail.tsx

```diff
@@ -117,9 +123,8 @@ const CurrentPaymentDetails = (props: ProductDetail) => {
       >
         <div
           css={css`
-            padding-bottom: ${subscription.card || subscription.mandate
-              ? space[3]
-              : 0}px;
+            padding-bottom: ${space[3]}px;
+
             ${minWidth.tablet} {
               margin: ${space[6]}px 0 0 0;
               padding: ${space[6]}px 0 0 0;
```

Comment:
The top margin and padding is being overridden on lines 134–135 by `margin: 0` and `padding: 0`.